DESCRIPTION

```diff
@@ -1,11 +1,10 @@
 Package: dashTable
 Title: Core Interactive Table Component for Dash
 Version: 4.5.1
-Authors @R: as.person(c(Chris Parmer <chris@plot.ly>))
-Description:  Dash DataTable is an interactive table component designed for designed for viewing, editing, and exploring large datasets. DataTable is rendered with standard, semantic HTML <table/> markup, which makes it accessible, responsive, and easy to style. This component was written from scratch in React.js specifically for the Dash community. Its API was designed to be ergonomic and its behavior is completely customizable through its properties.
+Description: Dash DataTable is an interactive table component designed for designed for viewing, editing, and exploring large datasets. DataTable is rendered with standard, semantic HTML <table/> markup, which makes it accessible, responsive, and easy to style. This component was written from scratch in React.js specifically for the Dash community. Its API was designed to be ergonomic and its behaviour is completely customizable through its properties.
```

Comment:
Shouldn't start with `Dash DataTable` for mysterious reasons

Reply:
```suggestion
Description: An interactive table component designed for designed for viewing, editing, and exploring large datasets, DataTable is rendered with standard, semantic HTML <table/> markup, which makes it accessible, responsive, and easy to style. This component was written from scratch in React.js specifically for the Dash community. Its API was designed to be ergonomic and its behaviour is completely customizable through its properties.
```